modules/javafx.graphics/src/main/java/com/sun/prism/es2/ES2SwapChain.java

```diff
@@ -190,7 +190,11 @@ public boolean present() {
 
     @Override
     public ES2Graphics createGraphics() {
-        if (drawable.getNativeWindow() != pState.getNativeWindow()) {
+        if (drawable == null || drawable.getNativeWindow() != pState.getNativeWindow()) {
```

Comment:
The null check wasn't here before, and it doesn't seem necessary as `createGLDrawable()` does not return `null`.

Reply:
But drawable can now be disposed and set to null. If that happens and `createGraphics` is called again it will end in a NPE.

Reply:
The NPE would be a good thing then, as no method should be called on a disposed object (except for `dispose`).

Reply:
Makes sense. I rolled back the null check.